runtime/queries/markdoc/highlights.scm

```diff
@@ -0,0 +1,16 @@
+tag_name: (identifier) @tag
+(tag_self_closing "/" @tag)
+(tag_close "/" @tag)
+([(tag_start) (tag_end) "="] @tag)
+(attribute [key : (identifier)] @attribute)
+(attribute [shorthand : (identifier)]  @attribute)
+(variable [variable : (identifier) (variable_sigil)] @variable)
+(variable_tail property : (identifier) @property)
```

Comment:
`property` maybe is `attribute` or `variable.other.member`


- https://docs.helix-editor.com/themes.html#syntax-highlighting